platforms/jvm/groovy-compiler-worker/build.gradle.kts

```diff
@@ -0,0 +1,33 @@
+plugins {
+    id("gradlebuild.distribution.api-java")
+}
+
+description = "Classes required to implement Groovy compiler workers. " +
```

Comment:
Do we need to ensure our dependencies don't pull groovy into our runtime classpath?